scripts/copyright-manager.ts

```diff
@@ -0,0 +1,87 @@
+import fs from 'fs-extra';
```

Comment:
Does the checker code itself need a copyright notice?
Is `manager` the right name for this file?

Reply:
Yes, it should include a copyright notice as well.
Since this tool can both check and fix copyright notices, I chose the name `manager`. That said, I don't have a strong preference and am open to alternative name suggestions.